app/ai/voice/agents/breeze_buddy/template/transformation_function/utils.py

```
@@ -210,3 +248,49 @@ def expand_shorthand(value: str) -> str:
+            fields if isinstance(fields, dict) else {k: _humanize_key(k) for k in item}
+        )
+        parts = []
+        for key, label in labels.items():
+            value = item.get(key, "")
+            # transform name (a string from the schema) -> the function in this module
+            transform = globals().get(transforms.get(key) or "")
+            if callable(transform) and value != "":
+                try:
+                    value = transform(value)
+                except Exception:
+                    pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)